Section 3.md

```diff
@@ -83,9 +83,9 @@ A colonial start-up, O'Neill Manufacturing meets the demands of a growing commun
 
 - **Xion Industrial**
 
-Xion was a company founded long ago, originally producing industrial equipment and vehicles, from drills to mechs. It only came to become the company we know it as today during the robotics boom of the 24th century, constructing drones for many industrial applications. They expanded at an incredibly rapid pace, quickly becoming a well-known name across Sol, and being a major contributor to the replacement of many human workers by robots.
+Xion was founded in the mid 23rd century, originally manufacturing industrial equipment, but during the Robotics Boom of the 24th Century, Xion reinvented itself into the company we see today- constructing a vast variety of different industrial drone tech. They quickly grew in size and power, becoming a well known name in Sol and paving the way for the robotic replacement of human workers in certain industrial fields.
 
-These days, they�re much less powerful than they were then, as other, stronger corporations progressed forward, using newer innovations such as positronics, whilst Xion sits back and continues to mass-produce cheap drones. In more recent years, they�ve introduced a line of prosthetics with a heavy focus on function over form, again being primarily for industrial work.
+These days, they're a far less powerful corporation then they were back then. Rather than moving forward with the newer innovations such as positronic technology, Xion continues to mass-produce their cheap drones. In more recent years, they've introduced a new line of prosthetic equipment who's primary focus is on function over design in the industrial workforce.
```

Comment:
*Today, they are far less powerful than they were then. 

*introduced a new line of prosthetic equipment, which prioritizes function over design, for use in the industrial sector.